src/js/electron/brim-start.test.ts

```diff
@@ -42,8 +42,14 @@ test("start opens default windows and in correct focus order", async () => {
   await brim.start()
   expect(brim.windows.count()).toBe(2)
   const windows = brim.windows.getAll()
```

Comment:
We could change this to be `windows.getVisible()`. That would remove the hidden window from the results.

Reply:
I thought about that, but I decided to keep the test as is so that it checks the behavior we want (hidden window should never have the most recent lastFocused timestamp)